features/admin.ai.v1/components/branding-card.tsx

```diff
@@ -0,0 +1,185 @@
+
+    const { isGeneratingBranding } = useAIBrandingPreference();
+
+    const generateAIBrandingPreference: GenerateAIBrandingPreferenceFunc = useGenerateAIBrandingPreference();
+
+    const handleExpandClick = () => {
+        setBannerState(BannerState.Input);
+    };
+
+    const handleCollapseClick = () => {
+        setBannerState(BannerState.Collapsed);
+    };
+
+    const handleGenerateClick = async () => {
+        await generateAIBrandingPreference(websiteUrl, "carbon.super");
+        setBannerState(BannerState.Collapsed);
+    };
+
+    return (
+        <>
+            <div
+                className={
+                    classNames(
+                        "branding-card",
+                        {
+                            hidden: isGeneratingBranding
+                        },
+                        "branding-card-banner-full"
+                    )
+                }>
+
+                { bannerState === BannerState.Full && (
+                    <Segment
+                        basic
+                    >
+                        <div className="branding-card-banner-full-heading">
+                            <div>
+                                <Header as="h3">{ t("branding:ai.banner.full.heading") }</Header>
+                                <p>{ t("branding:ai.banner.full.subHeading") }</p>
+                            </div>
+                            <Button onClick={ handleExpandClick } color="secondary" variant="outlined">
+                                <GenericIcon
+                                    icon={ AIIcon }
+                                    className="branding-card-banner-full-button"
+                                />
+                                { t("branding:ai.banner.full.button") }
+                            </Button>
+                        </div>
+                    </Segment>
+                ) }
+                { bannerState === BannerState.Input && (
+                    <Segment>
+                        <div className="branding-card-banner-input">
+                            <Icon
+                                name="dropdown"
+                                onClick={ handleCollapseClick }
+                                className="branding-card-banner-input-icon"
+                            />
+
+                            <div className="branding-card-banner-input-content">
+                                <div>
+                                    <Header as="h3" className="branding-card-banner-input-heading">
+                                        { t("branding:ai.banner.input.heading") }
+                                    </Header>
+                                    <p>{ t("branding:ai.banner.input.subHeading") }
+                                        <DocumentationLink
+                                            link={ "develop.applications.editApplication.asgardeoTryitApplication" +
```

Comment:
Check this link.

Reply:
added a dummy path with https://github.com/wso2/identity-apps/pull/5961/commits/d11ff93e5cc9790681d68829e1ce06b04e1dbfde temporarily.